spec/spec_helper.rb

```diff
@@ -274,8 +298,23 @@ def set_local_repository_format_to_future_version()
 end
 
 def set_remote_repository_format_to_future_version()
-  version_file = File.join(FIG_REMOTE_DIR, Fig::Repository::VERSION_FILE_NAME)
-  FileUtils.mkdir_p(FIG_REMOTE_DIR)
+  # Set future version in legacy remote dir
+  #version_file = File.join(FIG_REMOTE_DIR, Fig::Repository::VERSION_FILE_NAME)
+  #FileUtils.mkdir_p(FIG_REMOTE_DIR)
+  #File.open(version_file, 'w') {
+  #  |handle| handle.write(Fig::Repository::REMOTE_VERSION_SUPPORTED + 1)
+  #}
```

Comment:
This can probably go. It would be easy to reconstruct since it's the same as the code below but with the variable that's gone.